Adds in example of using TourFilters to disable certain tours (#2478)
umbraco/UmbracoDocs: Extending/Backoffice-Tours/index.md

### customProperties

A JSON object that is passed to the scope of a custom step view, so you can use this data in your view with `$scope.model.currentStep.customProperties`.

## How to filter/disable tours being shown

> It is possible to hide/disable tours using a C# composer by adding to the TourFilters collection. Here is an example of disabling all the CMS core tours based on the alias.

Reviewer: You say it's an example for disabling tours based on alias, right?
In the code example, it's also shown how to disable based on file and based on package - or am I completely off base here? 🤔

Reviewer: Right! Just noticed your comment 😅 🙈 

So, wouldn't it make sense to mention that in the text as well? That there are 3 different ways to filter out the Tours? 🤔

> It is possible to hide/disable tours using a C# composer by adding to the TourFilters collection. Here is an example of disabling all the CMS core tours based on the alias.

Author: ```suggestion
It is possible to hide/disable tours using a C# composer by adding to the TourFilters collection. Here is an example of disabling all the CMS core tours based on the alias., along with showing approaches how you could filter out tours by its JSON filename or by any tour JSON files coming from an App_Plugin plugin/package.
```

A JSON object that is passed to the scope of a custom step view, so you can use this data in your view with `$scope.model.currentStep.customProperties`.

## How to filter/disable tours being shown

> It is possible to hide/disable tours using a C# composer by adding to the TourFilters collection. Here is an example of disabling all the CMS core tours based on the alias., along with showing approaches how you could filter out tours by its JSON filename or by any tour JSON files coming from an App_Plugin plugin/package.

Reviewer: ```suggestion
It is possible to hide/disable tours using a C# composer by adding to the TourFilters collection.

Here is an example of disabling all the CMS core tours based on the alias, along with examples on how you could filter out tours by its JSON filename and how to disable tours from specific packages.
```
